.azure-pipelines/linux-CI-keras-applications-nightly.yml

```diff
@@ -23,13 +23,13 @@ jobs:
 
       Python36:
         python.version: '3.6'
-        ONNX_PATH: onnx==1.5.0
+        ONNX_PATH: onnx==1.6.0
```

Comment:
let's keep this to run some tests against opset 10

Reply:
Sure.